src/plugins/news/components/create-article-modal/Form.jsx

```diff
@@ -1,36 +1,54 @@
+import get from 'lodash/get';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { reduxForm, Field } from 'redux-form';
 import TextField from '@triniti/cms/components/text-field';
 import { Card, CardBody } from '@triniti/admin-ui-plugin/components';
+import { formRules } from '@triniti/cms/plugins/news/constants';
 
-const Form = ({ onBlurSlug: handleBlurSlug, onKeyDown: handleKeyDown }) => (
-  <Card>
-    <CardBody indent>
-      <Field
-        component={TextField}
-        label="Title"
-        name="title"
-        onKeyDown={handleKeyDown}
-        placeholder="enter title"
-        type="text"
-      />
-      <Field
-        component={TextField}
-        label="Slug"
-        name="slug"
-        onBlur={handleBlurSlug}
-        onKeyDown={handleKeyDown}
-        placeholder="enter slug"
-        type="text"
-      />
-    </CardBody>
-  </Card>
-);
+const Form = ({ formValues, onBlurSlug: handleBlurSlug, onKeyDown: handleKeyDown }) => {
+  const { TITLE_LENGTH_LIMIT } = formRules;
```

Comment:
@bhsiao55 @arunkarnati potentially a tiny issue here - the ticket is not clear, the title says the warning should show when the title _exceeds_ 75 but the description says _at_ 75. check w Sam

![Screen Shot 2021-01-29 at 5 32 23 PM](https://user-images.githubusercontent.com/6315388/106342978-08402e00-6258-11eb-9731-362d3c04d2fa.png)

Reply:
I will ask sam

Reply:
@efjacobson sam confirmed at 75